Sources/Stats.php

```diff
@@ -642,8 +642,8 @@ function DisplayStats()
 				'year' => $row_months['stats_year']
 			),
 			'href' => $scripturl . '?action=stats;' . ($expanded ? 'collapse' : 'expand') . '=' . $ID_MONTH . '#m' . $ID_MONTH,
-			'link' => '<a href="' . $scripturl . '?action=stats;' . ($expanded ? 'collapse' : 'expand') . '=' . $ID_MONTH . '#m' . $ID_MONTH . '">' . $txt['months'][(int) $row_months['stats_month']] . ' ' . $row_months['stats_year'] . '</a>',
-			'month' => $txt['months'][(int) $row_months['stats_month']],
+			'link' => '<a href="' . $scripturl . '?action=stats;' . ($expanded ? 'collapse' : 'expand') . '=' . $ID_MONTH . '#m' . $ID_MONTH . '">' . $txt['months_titles'][(int) $row_months['stats_month']] . ' ' . $row_months['stats_year'] . '</a>',
+			'month' => $txt['months_titles'][(int) $row_months['stats_month']],
 			'year' => $row_months['stats_year'],
```

Comment:
@d3vcho What is the difference between month_titles and months?  Seems like it contains the same info.

Reply:
Not for all languages. For example, in Ukrainian it contains various arrays (different declension). It helps to use correct forms in various forum's areas (Stats, Calendar, etc):

```
$txt['months'] = array(1 => 'Січня', 'Лютого', 'Березня', 'Квітня', 'Травня', 'Червня', 'Липня', 'Серпня', 'Вересня', 'Жовтня', 'Листопада', 'Грудня');
$txt['months_titles'] = array(1 => 'Січень', 'Лютий', 'Березень', 'Квітень', 'Травень', 'Червень', 'Липень', 'Серпень', 'Вересень', 'Жовтень', 'Листопад', 'Грудень');
```